game/src/core/hud/TextDialog.java

```diff
@@ -1,59 +1,27 @@
 package core.hud;
 
-import com.badlogic.gdx.scenes.scene2d.ui.*;
+import com.badlogic.gdx.scenes.scene2d.ui.Dialog;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
 
-import quizquestion.QuizQuestion;
+import java.util.function.BiFunction;
 
 /** Contains Constructor, which immediately creates the dialogue including all its elements. */
```

Comment:
das eine klasse in jaca einen ctor hat, ist eigentlich nicht so ungewöhnlich, dass es in die klassen-javadoc mit rein müsste ;)

mich würde eher interessieren: was macht die klasse, wozu ist sie da, wie benutze ich sie?